python/ray/serve/_private/autoscaling_policy.py

```diff
@@ -43,7 +43,12 @@ def calculate_desired_num_replicas(
     )
 
     # Multiply the distance to 1 by the smoothing ("gain") factor (default=1).
-    smoothed_error_ratio = 1 + ((error_ratio - 1) * autoscaling_config.smoothing_factor)
+    if error_ratio >= 1:
```

Comment:
Nitpick (non-blocker): Understood that if `error_ratio >= 1`, then the `smoothed_error_ratio` in line 51 will be calculated as a value larger than 1, meaning it's upscaling. Otherwise, it's calculated to a value smaller than 1, meaning it's a downscale. But would be nice to add some comment to explain how is this distinguished :)